### PR TITLE
Fix report tool tests

### PR DESCRIPTION
**Pull request by nocturnalastro**

### CODE REVIEW

File: pkg/compare/compare.go [hunk 1]
```diff
@@ -510,7 +511,7 @@ func (o *Options) Run() error {
 	// We will return exit code 1 in case there are differences between the reference CRs and cluster CRs.
 	// The differences can be differences found in specific CRs or the absence of CRs from the cluster.
 	if numDiffCRs != 0 || sum.NumMissing != 0 {
-		return exec.CodeExitError{Err: fmt.Errorf("there are differences between the cluster CRs and the reference CRs"), Code: 1}
+		return exec.CodeExitError{Err: errors.New(DiffsFoundMsg), Code: 1}
```
[the remark this answers is not in the document]
Comment (nocturnalastro) [reply pkg/compare/compare.go:514]: Yes the I used the new variable in the test fix, so it doesn't panic on an expected failure. 

https://github.com/openshift/kube-compare/pull/86/files#diff-259be8654dd520474eaeda133e5b9da4184f68c011f6ff51e40feb0515865aecR86

[the remark this answers is not in the document]
Comment (lack) [reply pkg/compare/compare.go:514]: Ah, missed that in the other file ;)

**Pull request merged**